### ISSUE TITLE
Move Convert methods back in Engine

### ISSUE THREAD

**Issue by alelom** [BHoM/Speckle_Toolkit#89]
#### Broken rules:

Tl;dr: converts are useful in the Engine in the Speckle_Toolkit case and should be moved back there.

Full description:

All Convert methods were moved in the Adapter project as a result of the latest compliance measures, intended to avoid exposing normally unneeded converts in the UI.

However, Speckle is as always a bit particular, and there are fringe cases where you might want to do the convert for certain objects in the UI, while explicitly avoiding the converts for others. For example, this allows to use the standard Speckle "Sender" component to upload native Rhino geometries together with BHoMObjects by separating them and avoiding having them to pass through a conversion phase all together. Complicated, but can be an extremely useful workaround in fringe situations.

#### Suggestions to restore compliance:

As above
